shell/shared/grain.js

```diff
@@ -126,13 +124,6 @@ if (Meteor.isServer) {
       promise.cancel();
     });
   });
-
-  function cleanupExpiredTokens() {
```

Comment:
The transient offer-template tokens are deleted in a separate check: https://github.com/sandstorm-io/sandstorm/blob/c47f4ab7e304933ba1b968f89fb4625a8af931b3/shell/server/hack-session.js#L368-L375